data_structures/heap/min_heap.py

```diff
@@ -0,0 +1,154 @@
+# Min head data structure
+# with decrease key functionality - in O(log(n)) time
+
+
+class Node:
+    def __init__(self, val, name):
```

Comment:
It will be more intuitive to change the order of the arguments to be __name, value__ so it is like a Python dictionary __key, value__